Sources/HTTPParser.swift

```diff
@@ -51,8 +51,8 @@ class HTTPParser {
     var components = top.split("\r\n")
     let requestLine = components.removeFirst()
     components.removeLast()
-    let responseComponents = requestLine.split(" ", maxSplit: 3)
```

Comment:
I'm not sure this is correct?

It should be "VERSION STATUS-CODE STATUS-REASON" where status reason may contain spaces?